react/components/BuyButton/index.js

```diff
@@ -8,30 +8,28 @@ import { injectIntl, intlShape } from 'react-intl'
 import ADD_TO_CART_MUTATION from './mutations/addToCartMutation.gql'
 import ORDER_FORM_QUERY from './queries/orderFormQuery.gql'
 
-const EVENT_SUCCESS = 'item:add'
-const EVENT_ERROR = 'message:error'
+const CONSTANTS = {
+  EVENT_SUCCESS: 'item:add',
+  EVENT_ERROR: 'message:error',
+  ERROR_MESSAGE_ID: 'buybutton.add-failure',
+  CHECKOUT_URL: '/checkout/#/cart',
```

Comment:
I don't think that's a responsibility of buy-button make the redirect. He should send the event that someone click and the component who have to decide if redirect or not.

Reply:
But, we don't have an event center that handles incoming events to decide what action to make.